frontend/actions/release.ts

```diff
@@ -26,3 +26,11 @@ export async function postRelease(release: Partial<ReleaseInterface>, modelId: s
     body: JSON.stringify(release),
   })
 }
+
+export async function postArtefact(artefact: File, modelId: string, name: string, mime: string) {
```

Comment:
This should be called 'postFile'.  In general, 'artefacts' include both docker images and files.  Whereas files are just files!